internal/actor/certificate_provider_provided.go

```diff
@@ -31,6 +31,8 @@ type CertificateProviderProvidedDetails struct {
 	Tasks CertificateProviderTasks
 	// ContactLanguagePreference is the language the certificate provider prefers to receive notifications in
 	ContactLanguagePreference localize.Lang
+	// DonorChannel is the way the donor is applying for the LPA (online or paper)
+	DonorChannel Channel
```

Comment:
I was on the fence between adding this to the CP vs persisting a paper LPA in dynamo - having a shadow of the LPA store LPA felt like it could lead to some data inconsistencies but this means in a future PR we'll need to point all LPA gets for CPs (and eventually attorneys with a paper donor) to the lpa store rather than dynamo.

Reply:
maybe this ticket should be done after the one I'm currently doing, because that will be the case after I'm done